### DISCUSSION TITLE
How can I increase the clock speed in simulation mode?

### DISCUSSION THREAD

**Discussion by Alavins** [logisim-evolution/logisim-evolution#1894] [Q&A] [answered]
I use the latest version and develop circuits using large amounts of memory and RGB-video modules, but in simulation mode the clock frequency does not rise above 4 kHz (nevertheless, in the simulation settings I set the highest clock frequency - 2 MHz). I changed the settings in the graphics acceleration window, but the clock speed did not increase.

**Comment by maehne** [comment, marked as answer]
Simulation performance is limited by the redrawing of the GUI window at each simulation step. See issue #786 for details. I am closing this issue as a duplicate issue.
